dune-project

```diff
@@ -1,3 +1,432 @@
 (lang dune 2.7)
```

Comment:
I guess this in combination with a `depends` below that requires `dune (>= 3.2.0)` is what producing those weird `"dune" {>= "2.7" & >= "3.2.0"}` constraints?

Reply:
yes, I think so.